lyaemu/coarse_grid.py

```diff
@@ -253,6 +255,14 @@ def get_nsample_params(self):
         """Get the number of sparse parameters, those sampled by simulations."""
         return np.shape(self.param_limits)[0]
 
+    def _get_pv(self, pp, myspec):
+        """ Helper function to get a single pdf vector """
+        di = self.get_outdir(pp, strz=3)
+        if not os.path.exists(di):
+            di = self.get_outdir(pp, strz=2)
+        single_flux_pdf = myspec.get_snapshot_list(base=di)
+        return single_flux_pdf
+
```

Comment:
It is probably easier to just rename the variables in _get_fv to something generic. The functions are the same and if they are reused it means that fixes to one propagate to the others.